{"name": "test_openanaccount_a1_error_3", "status": "passed", "description": "账户开户失败的用例", "attachments": [{"name": "log", "source": "219878b1-f294-41f5-a18d-8d2d14cd7bbb-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "dc8595c9-2fd4-423d-9e94-b18352c02bf1-attachment.txt", "type": "text/plain"}], "start": 1617349378620, "stop": 1617349380915, "uuid": "04264fd8-e3bd-4298-975d-c1560e895dad", "historyId": "5c882184ce7d4ece271057c7e0c1cd4a", "testCaseId": "84e3746db8637a693ecefdedfc13aef9", "fullName": "tests.test_a8_the_customer_account.TestTheCustomerAccount#test_openanaccount_a1_error_3", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a8_the_customer_account"}, {"name": "subSuite", "value": "TestTheCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a8_the_customer_account"}]}